VideoLocker/res/layout/edx_spinner_item.xml

```diff
@@ -12,4 +12,5 @@
     android:singleLine="true"
     android:textColor="@color/edx_grayscale_neutral_dark"
     android:textSize="@dimen/edx_x_small"
+    android:textDirection="locale"
```

Comment:
I don't think the textviews should be forced to the `locale`, since the user may be viewing content that is not in their native language, and I don't think we should override it's text direction.